.github/workflows/ci.yaml

```diff
@@ -0,0 +1,98 @@
+---
+name: ci
+on: pull_request
+jobs:
+  yamllint:
```

Comment:
yamllint job can be simplified https://github.com/networkservicemesh/cmd-template/blob/master/.github/workflows/ci.yaml#L9